com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -887,6 +905,12 @@ static void UpgradeAsset(UniversalRenderPipelineAsset asset)
 
                 asset.k_AssetPreviousVersion = 5;
             }
+
+            if (asset.k_AssetPreviousVersion < 7)
+            {
+                asset.postProcessData = PostProcessData.GetDefaultPostProcessData();
+                asset.k_AssetPreviousVersion = 7;
+            }
```

Comment:
After having a good look at this, it seems there is some work to be done for the upgrader, for now the only tweak I would suggest is to add `EditorUtility.SetDirty(asset);` after these two upgraders to make sure we are dirtying users asset files, otherwise they may continue to upgrade their assets without saving.
Then it's green from me and we need to look at making this asset upgrader more along the lines of the material upgrader in a later PR.